### PR TITLE
Updating Readme to make writePoints function work

### PR DESCRIPTION
**Pull request by manorie**
Hello,

I have tried this example as it is for now. Encountered an error that AddPoint function expects only one argument. Checked projects GoDoc and updated the example in your error handling style. It works like this.

###### Required for all non-trivial PRs
- [X] Rebased/mergable
- [X] Tests pass
- [ ] CHANGELOG.md updated
(seemed unnecessary as it is related to the README example function fix)
- [X] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

###### Required only if applicable
_You can erase any checkboxes below this note if they are not applicable to your Pull Request._
- [ ] [InfluxQL Spec](https://github.com/influxdata/influxdb/blob/master/influxql/README.md) updated
- [ ] Provide example syntax
- [ ] Update man page when modifying a command
- [ ] [InfluxData Documentation](https://github.com/influxdata/docs.influxdata.com): issue filed or pull request submitted \<link to issue or pull request\>

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -142,12 +142,19 @@ func writePoints(clnt client.Client) {
...
+		pt, err := client.NewPoint(
 			"cpu_usage",
 			tags,
 			fields,
 			time.Now(),
-		))
+		)
+		
+		if err != nil {
+			log.Fatalln("Error: ", err)
```
Comment (e-dard) [inline client/README.md]: This can just be `log.Fatal(err)`.

Comment (manorie) [inline client/README.md]: Yes this bothers me also, I just made it to be consistent with previous error handlings in the same sniplet. Let me clear it.

Comment (e-dard) [inline client/README.md]: @manorie you're welcome to clean the rest of them up too. 😄

**Comment by manorie** [comment]
As I have seen you are open to improvements @e-dard I changed log functions, added small improvements to make it more readable and clean (imo). Hope you like it.

File: client/README.md [hunk 2]
```diff
@@ -119,16 +119,21 @@ NOTE: You can specify a RetentionPolicy as part of the batch points. If not
...
+	bp, err := client.NewBatchPoints(client.BatchPointsConfig{
 		Database:  "systemstats",
 		Precision: "us",
 	})
+	if err != nil {
+		log.Fatal(err)
+	}
 
 	for i := 0; i < sampleSize; i++ {
+		rand.Seed(uint64(i))
```
Comment (e-dard) [inline client/README.md]: What is this line for?

Comment (manorie) [inline client/README.md]: It is a good catch. I wanted to give a true randomness here by incrementing the seed with iterator. Every time this function runs, will create a new graph. But the mistake I made here is to get confused about how to do it, I think taking the seed with the way of the last commit will change the sequence in every run.

**Comment by e-dard** [comment]
This looks great @manorie. Thanks for your work here!

**Comment by manorie** [comment]
You are welcome @e-dard . I will check open issues to see if I can contribute more in my spare time. Hope I can contribute more.

**Pull request merged**